src/App.jsx

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
+import { Sum } from './components/Sum';
 import './App.css';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum a={2} b={3} />
+    <Sum a={-5} b={5} />
+    <Sum a={10} b />
+    <Sum a b={5} />
+    <Sum a b />
```

Comment:
```suggestion
    <Sum a={10} />
    <Sum b={5} />
    <Sum />
```
pass prop without value will be true.
it should fix tests
![Screenshot from 2023-06-10 18-36-43](https://github.com/mate-academy/react_sum/assets/33029864/3b53f7c3-721b-481c-b853-cdf49fa4534f)